pkgs/development/python-modules/speechbrain/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, huggingface-hub
+, hyperpyyaml
+, joblib
+, lib
+, numpy
+, packaging
+, pythonOlder
+, sentencepiece
+, scipy
+, torchaudio
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "speechbrain";
+  version = "0.5.14";
+
```

Comment:
I set `format = "pyproject"` but it doesn't seem to complain about `setuptools` not being in `nativeBuildInputs`.
Should I leave it like this or add it nonetheless ?

Reply:
Perhaps something in the Python infrastructure falls back to `setuptools`, I'm not sure. I prefer to include it for clarity, but I'm not sure what is "correct".

Reply:
@SuperSandro2000 you have an opinion on this ?

Reply:
> Since this repo has a `pyproject.toml` file, can we add:

That's not the deciding factor. The project needs to use that file to build the package which it doesn't https://github.com/speechbrain/speechbrain/blob/develop/pyproject.toml

Reply:
I came to a different conclusion in https://github.com/NixOS/nixpkgs/pull/240234#discussion_r1245021614.